src/amo/sagas/reviews.js

```diff
@@ -70,7 +72,33 @@ function* handleReplyToReview(
   }
 }
 
+function* handleFlagReview(
```

Comment:
As mentioned, I don't really like flag as user-facing OR developer-facing terminology. I think using "abuse report" whenever possible is clearer and sets up expectations better. So I'd rather this be `handleReviewAbuseReport` or something similar.

Reply:
It's not just an abuse report. You "flag" a review so that an admin can inspect it. You might flag it for abuse but you might also flag it for other reasons.

Reply:
Yeah, I realised this after but I think maybe just "report" might be a better verb than "flag".